### PR TITLE
Publish YDB CLI 2.17.0 

### PR DESCRIPTION
**Pull request by galnat**
### Changelog entry 

Publish YDB CLI 2.17.0 with Release Notes

### Changelog category 

* Documentation (changelog entry is not required)

### Additional information
YDBDOCS-936

### CODE REVIEW

**Review by pnv1 — DISMISSED** [review]

File: ydb/docs/en/core/changelog-cli.md [hunk 1]
```diff
@@ -1,5 +1,19 @@
...
+Released on December 4, 2024. To update to version **2.17.0**, select the [Downloads](downloads/index.md#ydb-cli) section.
+
+### Features
+
+* Improved performance of parallel [importing data from the file system](./reference/ydb-cli/export-import/tools-restore.md) using the `{{ ydb-cli }} tools restore` command.
+* Added the `{{ ydb-cli }} debug ping` command for performance and connection debugging.
+
+### Bug fixes
+
+* Fixed a bug in the table schema created by the command `{{ ydb-cli }} workload tpch` where the `partsupp` table contained an incorrect list of key columns.
```
Comment (blinkov) [inline ydb/docs/en/core/changelog-cli.md:14]:
```suggestion
* Fixed a bug in the table schema created by the `{{ ydb-cli }} workload tpch` command where the `partsupp` table contained an incorrect list of key columns.
```

File: ydb/docs/en/core/changelog-cli.md [hunk 2]
```diff
@@ -1,5 +1,19 @@
 # {{ ydb-short-name }} CLI changelog
 
+## Version 2.17.0 {#2-17-0}
+
+Released on December 4, 2024. To update to version **2.17.0**, select the [Downloads](downloads/index.md#ydb-cli) section.
+
+### Features
+
+* Improved performance of parallel [importing data from the file system](./reference/ydb-cli/export-import/tools-restore.md) using the `{{ ydb-cli }} tools restore` command.
```
Comment (blinkov) [inline ydb/docs/en/core/changelog-cli.md:9]: We'd better use the same categories for cli and server, and this sounds more like performance improvement.

File: ydb/docs/en/core/changelog-cli.md [hunk 3]
```diff
@@ -1,5 +1,19 @@
...
+* Added the `{{ ydb-cli }} debug ping` command for performance and connection debugging.
+
+### Bug fixes
+
+* Fixed a bug in the table schema created by the command `{{ ydb-cli }} workload tpch` where the `partsupp` table contained an incorrect list of key columns.
+* Resolved an issue where the `{{ ydb-cli }} tools restore` command failed with the error "Too much data" if the maximum value of the `--upload-batchbytes` option (16 MB) was set.
```
Comment (blinkov) [inline ydb/docs/en/core/changelog-cli.md:15]:
```suggestion
* Resolved an issue where the `{{ ydb-cli }} tools restore` command failed with the error "Too much data" if the maximum value of the `--upload-batchbytes` option was set to 16 MB.
```
or more than 16 MB?

File: ydb/docs/en/core/changelog-cli.md (hunk 2 above)
Comment (galnat) [reply ydb/docs/en/core/changelog-cli.md]: Added performance section as in the server

**Review by fomichev3000 — APPROVED** [review]

**Pull request merged**